changelog/added-new stm32-targets.md

```diff
@@ -0,0 +1,2 @@
+Update the `STM32WB_Series.yaml` targets with `target-gen arm --filter STM32WBxx_DFP`
+Removed `stack_size` from all `flash_algorithms` files as the values are sometimes incorrect and are not needed for flashing operations.
```

Comment:
Just a formality, I don't think you need to mention the stack_size change - it's not necessarily important for users and the changelog will generally look better with one line per entry :)

Reply:
Addressed here: https://github.com/kquinsland/probe-rs/commit/38c6284ddb92f0d7463f6a37b4c69142b617ddca